pseudocode.md

```diff
@@ -1,6 +1,19 @@
 # Hangman (Pseudocode)
 
 ```
-Write pseudocode here, within the tripple backticks.
+get user input for the theme
```

Comment:
We can implement themes in our game. Think of how to do this and offer a pull request.